docs/projects/myo/setup.md

```diff
@@ -183,58 +173,54 @@ Copy this file into `workers/my-unity-project/Assets/Generated/Improbable.Gdk.Ge
 
 **Step 2.** Add package references to your `manifest.json`.
 
-Unity loads the packages that are declared in the `manifest.json` file into your Unity project. We will be side-loading the GDK packages.
+Unity loads the packages that are declared in the `manifest.json` file into your Unity project. We will be including the GDK packages.
```

Comment:
```suggestion
Unity loads the packages that are declared in the `manifest.json` file into your Unity project.
```

Just get rid of this imo. It was noted previously as it was exceptional rather than the norm